plugins/kafka/librdkafka

```diff
@@ -0,0 +1 @@
+Subproject commit 3e1babf4f26a7d12bbd272c1cdf4aa6a44000d4a
```

Comment:
At one point I had been using features found only on the master branch.  The code changed though, and that may not be the case any longer.  I'll see if I can use one of their release branches.